src/Features/ERDDAP/Platform/Observations/Table/table.tsx

```diff
@@ -1,14 +1,17 @@
 /**
  * Current observations table component
  */
-import * as React from "react"
+import React from "react"
```

Comment:
Codacy found an issue: ['React' was used before it was defined.](https://app.codacy.com/manual/gmri/Neracoos-1-Buoy-App/pullRequest?prid=6166252)